features/src/rapids-build-utils/opt/rapids-build-utils/bin/make-pip-dependencies.sh

```diff
@@ -59,6 +62,30 @@ make_pip_dependencies() {
     local python_version="${PYTHON_VERSION:-$(python3 --version 2>&1 | cut -d' ' -f2)}";
     python_version="$(cut -d'.' -f3 --complement <<< "${python_version}")";
 
+    # Why default to cuda_suffixed=true?
+    #
+    # Projects that depend on different pip libraries across different CUDA versions
+    # (e.g. 'cudf' only depending on 'pynvjitlink' from CUDA 12.0 onwards), split up their
+    # dependency lists with 'cuda_suffixed={true,false}'.
+    #
+    # Here we want the suffixed versions (like 'pynvjitlink-cu12').
+    #
+    # It's ok for other RAPIDS libraries to end up in this list (like 'rmm-cu12')... in builds
+    # where those are also being built in the devcontainer, they'll be filtered out via
+    # inclusion in the 'pip_noinstall' list below.
+    local -a _matrix_selectors=(
+        arch="$(uname -m)"
+        cuda="${cuda_version}"
+        cuda_suffixed=true
+        py="${python_version}"
+    );
+
+    # add extra arguments (if there are conflicts, e.g. 'py=3.10;py=3.11', it's fine... the last one will win)
+    local ent; for ent in "${matrix_entry[@]}"; do
+        _matrix_selectors+=("${ent}")
+    done
```

Comment:
```suggestion
    test ${#matrix_entry[@]} -gt 0 && _matrix_selectors+=("${matrix_entry[@]}");
```

Reply:
ok yeah this is cleaner (and matches all the other arguments), thank you